backend/graphql_v1/universal_codes/src/types/drug_interaction.rs

```diff
@@ -1,15 +1,3 @@
-/*
-type DrugInteractionType {
-  description: String!
-  name: String!
-  rxcui: String!
-  severity: String!
-  source: String!
-}
-*/
-
-//TODO: Implement this properly, right now this is just stub
```

Comment:
This is just a tidy up I saw.
Drug interactions aren't working but I think this is going to remain a stub for now anyway so didn't want the lingering todo...